getmoda.io.domain.json

```diff
@@ -0,0 +1,43 @@
+{
+    "providerId": "getmoda.io",
+    "providerName": "Moda",
+    "serviceId": "domain",
+    "serviceName": "Moda Domain",
+    "logoUrl": "https://userconfig.getmoda.io/moda-logo/moda-icon.svg",
+    "description": "This is a template to enable domain verification on Moda",
+    "variableDescription": "cname_host is the mail CNAME. cname_points is the value to which mail CNAME points to. dkim_points and dkim2_points are for the DKIM records ",
+    "syncRedirectDomain": "app.getmoda.io, moda-ui-dev.getmoda.io, moda-ui-stg.getmoda.io",
+    "syncPubKeyDomain": "getmoda.io",
+    "version": 1,
+    "records":
+    [
+        {
+            "groupId": "moda-cname",
+            "type": "CNAME",
+            "host": "%cname_host%",
+            "pointsTo": "%cname_points%",
+            "ttl": 600
+        },
+        {
+            "groupId": "moda-dkim",
+            "type": "CNAME",
+            "host": "md._domainkey",
+            "pointsTo": "md.domainkey.%dkim_points%",
+            "ttl": 600
+        },
+        {
+            "groupId": "moda-dkim2",
+            "type": "CNAME",
+            "host": "md2._domainkey",
+            "pointsTo": "md2.domainkey.%dkim2_points%",
+            "ttl": 600
+        },
+        {
+            "groupId": "moda-dmarc",
+            "type": "TXT",
+            "host": "_dmarc",
+            "data": "v=DMARC1; p=none; rua=mailto:dmarc@getmoda.io",
```

Comment:
Just realised one more thing. For DMARC you likely want to prevent that there is more than one DMARC record on the domain. For this you may want to use `txtConflictMatchingMode`=`Prefix` and `txtConfictMatchingPrefix`=`v=DMARC1`
https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#record-types-conflicts

It's not supported by all DNS providers, however it would be helpful for those who do.

Reply:
@pawel-kow Thank you. I've made the changes.